tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/RcmBase.cs

```diff
@@ -14,13 +16,17 @@ public class RcmBase : AspNetBase
     protected const string LogFileNamePrefix = "dotnet-tracer-managed-";
 
 #pragma warning disable SA1401
+    protected readonly string logDirectory;
```

Comment:
nit: I think stylecop pushes `protected` properties to be capitalized
```suggestion
    protected readonly string LogDirectory;
```

Reply:
I tried but I still get a stylecop error `SA1401`
![image](https://user-images.githubusercontent.com/8353486/194553586-146544ae-6a8c-462d-9cba-1972c7b2d419.png)
I don't know why cause the field is used in the children classes

Reply:
Maybe not as pretty, but you could also write the property like this:
```suggestion
    protected string LogDirectory => Path.Combine(DatadogLogging.GetLogDirectory(), $"{GetTestName()}Logs");
```

Reply:
thanks, fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3320/commits/dbd302f4afbca2db06bdbc0ee3fbacdfb42c8a01